validator/errors/errors.py

```diff
@@ -0,0 +1,22 @@
+"""Custom Errors for Nestor Config Validator"""
+
+
+class InvalidTargetPathError(Exception):
+    """Exception raised when the target path (NESTOR_TARGET_PATH) env is not set"""
+
+    def __init__(self, target_path: str):
+        super().__init__(
+            f"The environment variable NESTOR_TARGET_PATH is not set. Current value: ${target_path}"
+        )
+
+
+class ConfigurationNotValidError(Exception):
+    """Exception raised when a configuration is invalid"""
+
+    def __init__(self, validation_errors):
+        self.validation_errors = validation_errors
+        self.message = "Please fix the configuration with the help of the validationErrors property"
+        super().__init__(self.validation_errors)
```

Comment:
I would have expected `super` to be called with a message. What is the type of `validation_errors` ?

Reply:
This will be an array with a specific error for each file, I will specify it in the next PR 👍